Tests/Opc.Ua.Core.Tests/Types/Utils/UtilTests.cs

```diff
@@ -87,6 +87,68 @@ public void ToHexBigEndian()
             var hexutil = Utils.ToHexString(blob, true);
             Assert.AreEqual(hex, hexutil);
         }
+
+
+        /// <summary>
+        /// 
```

Comment:
Could you fill in the summary fields and explain what the test does?

Reply:
added summaries for each unit test